app/frontend/src/assets/scss/style.scss

```diff
@@ -114,41 +114,17 @@ a, .v-tab {
   }
 }
 
-// Cards
-.silv-info-card {
-  background-color: $bcgov-grey !important;
-  .v-icon {
-    color: $bcgov-blue-secondary !important;
-    margin-right: .5em;
-  }
-  ul {
-    padding-left: 4em !important;
-  }
-}
-
-.silv-contact-card {
-  background-color: $bcgov-blue-secondary !important;
-  h3 {
-    color: white !important;
-  }
+.question-series {
   p {
-    color: white !important;
+    font-size: 16px;
   }
-  .v-icon {
-    color: white !important;
-    margin-right: .5em;
+  .question-head {
```

Comment:
a lot of the vuetify measurements are in pixels, so i dont see a problem with this